zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/CamundaExporter.java

```diff
@@ -65,9 +63,9 @@ public CamundaExporter(final ExporterResourceProvider provider) {
 
   @Override
   public void configure(final Context context) {
-    configuration =
-        context.getConfiguration().instantiate(ElasticsearchExporterConfiguration.class);
+    configuration = context.getConfiguration().instantiate(ExporterConfiguration.class);
     provider.init(configuration);
+    createClientAdapter();
```

Comment:
maybe a better name `setClientAdapter`? Or you can do a refactor with the enhanced switch to return values/throw and do a `clientAdapater = createClientAdapter()`